gneiss/plot/_radial.py

```diff
@@ -0,0 +1,144 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+from gneiss.plot._dendrogram import UnrootedDendrogram
+
+try:
+    from bokeh.models.glyphs import Circle, Segment
+    from bokeh.models import ColumnDataSource, DataRange1d, Plot
+    from bokeh.models import HoverTool, BoxZoomTool, ResetTool
+except ImportError:
```

Comment:
I'm trying to avoid my dependencies blowing up - so all of the visualization code that I'm using will be based on optional dependencies.